app/services/imports/case_logs_import_service.rb

```diff
@@ -162,7 +177,11 @@ def safe_string_as_integer(xml_doc, attribute)
     # Safe: A string that represents only a decimal (or empty/nil)
     def safe_string_as_decimal(xml_doc, attribute)
       str = field_value(xml_doc, "xmlns", attribute)
-      BigDecimal(str, exception: false)
+      if str.blank?
+        BigDecimal("0.0")
+      else
+        BigDecimal(str, exception: false)
+      end
```

Comment:
Charges are only 0 if one of them is set

`  if %i[brent scharge pscharge supcharg].any? { |f| public_send(f).present? }
      self.brent ||= 0
      self.scharge ||= 0
      self.pscharge ||= 0
      self.supcharg ||= 0
      self.tcharge = brent.to_f + scharge.to_f + pscharge.to_f + supcharg.to_f
    end`